src/schemas/Helpdesk/index.ts

```diff
@@ -128,16 +124,11 @@ const createDisabledResolvers = (reCaptchaClient: ReCaptchaClient) => ({
         `FEATURE_HELP_DESK_ENABLED is ${FEATURE_HELP_DESK_ENABLED} in this instance of the gateway`,
       );
     }, reCaptchaClient),
-    createJiraTicket: () => {
-      throw new ApolloError(
-        `FEATURE_HELP_DESK_ENABLED is ${FEATURE_HELP_DESK_ENABLED} in this instance of the gateway`,
-      );
-    },
   },
 });
 
 export default async () => {
-  const reCaptchaClient = await createReCaptchaClient(); // this client is created regardless of FEATURE_HELP_DESK_ENABLED so its vault integration can be tested in dev environments
+  const reCaptchaClient = FEATURE_HELP_DESK_ENABLED ? await createReCaptchaClient(): await createStubReCaptchaClient(); // this client is created regardless of FEATURE_HELP_DESK_ENABLED so its vault integration can be tested in dev environments
```

Comment:
If we use a stub client then it's not really testable in dev anyway so the comment would be irrelevant. 

Basically the situation is, outside of prod, we don't have a JIRA to integrate with, so we left `FEATURE_HELP_DESK_ENABLED` out in other environments. But there, we still wanted to be able to test out reCaptcha, which is why reCaptcha integration was left in place in the `createDisabledResolvers`.

I think it'd be better to introduce a `DEV_RECAPCHA_DISABLED` flag for local dev and use it here so we retain the ability to test reCaptcha. No need to update any existing infra stuff because it'd be `false` by default. We can also update the error message thrown by `createReCaptchaClient` to mention this new flag... What do you think?

Reply:
yeah that makes sense